station/app/api/api_v1/endpoints/airflow.py

```diff
@@ -16,7 +61,7 @@ def get_airflow_run_information(run_id: str, dag_id: str):
     return run_info
 
 
-@router.get("/getAirflowTaskLog/{dag_id}/{run_id}/{task_id}/{task_try_number}", response_model=AirflowTaskLog)
+@router.get("/task/log/{dag_id}/{run_id}/{task_id}/{task_try_number}", response_model=AirflowTaskLog)
```

Comment:
`/logs/{dag_id}/{run_id}/{task_id}/{task_try_number}`

Maybe make task try number and use latest?